src/types/interfaces.ts

```diff
@@ -567,59 +575,7 @@ export interface IPersonDetails {
   chat: {
     jabberId: string;
   };
-}
-
-export interface ILogger {
```

Comment:
This is referenced in `doc/index.md`. We should change the interface there to match the interface being imported from gc-logger.